.github/workflows/ci.yml

```diff
@@ -65,6 +65,15 @@ jobs:
 
           # run the linter
           ./bin/golangci-lint run ./test/azure/ --build-tags=azure --timeout 5m0s
+      - name: setup terraform
+        uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_wrapper: false
+        id: azure_terraform_install
```

Comment:
If we are now installing terraform with this script, how were the go tests working before (since terratest just calls the terraform binary)?

Reply:
gah, terraform is on the agent by default.  I checked version and its running 0.13.3.  I am will remove this step as it is unnecessary.